modules/customize-certificates-understanding-default-router.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// authentication/certificates/replacing-default-ingress-certificate.adoc
+
+[id="understanding-default-ingress-{context}"]
+= Understanding the default ingress certificate
+
+By default {product-title} uses the Ingress Controller Operator to 
```

Comment:
There were discussions regarding this before I left, and it seems that the team has finalized on "Ingress Operator" (specifically when the operator is in use), or just "ingress" otherwise. I've fixed it here, and will open up a separate BZ to make these references consistent across the docs.